### ISSUE TITLE
Icon on sorting feature to show it is expandable

### ISSUE THREAD

**Issue by HSips** [huridocs/uwazi#954]
It has occurred to be when showing Uwazi here at the Centre for Human Rights that the sorting feature ,,see screenshot

![pretoria-1](https://cloud.githubusercontent.com/assets/22650647/25630190/f165a422-2f74-11e7-973a-af29f8f0eccb.png)

**Comment by danicatalan** [comment]
The problem we tried to solve here is to make a clearer interaction on the sorting. One of the things I noticed is before this design it was just a plain text. I tried to add some visuals, like an icon or an hover state (just like other buttons).

![default](https://user-images.githubusercontent.com/6364153/26925087-9eee0658-4c48-11e7-8cd2-31cc18f686ca.png)

![hover](https://user-images.githubusercontent.com/6364153/26924937-1647e184-4c48-11e7-8f16-da0096601a76.png)

Also I thing is useful to highlight the active attribute with a subtle background:

![active](https://user-images.githubusercontent.com/6364153/26924936-16425e1c-4c48-11e7-9119-fcaf77f4ef1b.png)

This is the recording video (not as good quality has the screenshots, sorry):

![img](http://g.recordit.co/NpapWirrq1.gif)
